docs/about/join.md

```diff
@@ -41,24 +45,33 @@ To apply for an account on the Wynton HPC cluster, please fill out the following
  
  <div class="form-group">
   <label for="email_address">UCSF email address</label> <span style="color: red">*</span><br>
-  <input type="email" id="email" name="email" autocomplete="email" placeholder="Enter email" size="30" required><br>
+  <input type="email" id="email" name="email" autocomplete="email" placeholder="Enter email" size="30" required><br> 
+  <small>(**External collaborators only: Please provide your preferred email address)</small>
  </div>
  
- <div class="form-group">
-  <label>UCSF affiliation</label> <span style="color: red">*</span><br>
+ <div class="form-group"> 
+  <label for="affiliation">UCSF affiliation</label> <span style="color: red">*</span><br>
```

Comment:
I think it's great that you've tried to add the `for` attribute to the HTML label tag.  I wasn't aware of it.  However, from <https://www.w3schools.com/tags/tag_label.asp>, it looks like it is meant to specify an `id` attribute but It looks like what you've specified, `"affiliation"`, matches the `name` attribute.  Are you sure what you've added, works?

Reply:
It worked, but I will change it to be consistent with the rest of the label tags

Reply:
What do you mean by "It worked"?

It's not clear to me how to map one label to a set of radio elements, but then I also know very little about this.  If you can figure it out, it'll probably improve the form. (e.g. accessibility)

Reply:
Updated code and validated